api/src/main/java/com/example/api/service/util/EmailService.java

```diff
@@ -0,0 +1,56 @@
+package com.example.api.service.util;
+
+import com.example.api.model.user.User;
+import com.mailjet.client.ClientOptions;
+import com.mailjet.client.MailjetClient;
+import com.mailjet.client.MailjetRequest;
+import com.mailjet.client.MailjetResponse;
+import com.mailjet.client.errors.MailjetException;
+import com.mailjet.client.errors.MailjetSocketTimeoutException;
+import com.mailjet.client.resource.Emailv31;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+@Transactional
+public class EmailService {
+    private final String API_KEY = "";
+    private final String SECRET_API_KEY = "";
+
+    public void sendEmail(User user, String subject, String message) {
+        MailjetClient client;
+        MailjetRequest request;
+        MailjetResponse response;
+        client = new MailjetClient(API_KEY, SECRET_API_KEY, new ClientOptions("v3.1"));
+        request = new MailjetRequest(Emailv31.resource)
+                .property(Emailv31.MESSAGES, new JSONArray()
+                        .put(new JSONObject()
+                                .put(Emailv31.Message.FROM, new JSONObject()
+                                        .put("Email", "wlodarskibartek2@gmail.com")
```

Comment:
Pytanie - czy ten adres email podawany tutaj do puta musi isnieć ? Bo jeżeli nie to weź walnij `messages-noreply@sys-chaos.com` xD.

Reply:
Jeżeli nie da się dać nieistniejącego to może zostaw jak jest na razie xD. Bez sensu zakładać gmaila tylko po to jak pewnie i tak nikt z tego nie skorzysta poza testami xD

Reply:
Potestowałem trochę i chyba trzeba mieć prawdziwy adres, więc zostawiam to tak